bin/update/deploy_base.py

```diff
@@ -133,7 +141,10 @@ def update_info(ctx):
 def peep_install(ctx):
     """Install things using peep."""
     with ctx.lcd(settings.SRC_DIR):
-        ctx.local('../venv/bin/peep install -r requirements/compiled.txt')
+        ctx.local(peep_install_cmd('prod'))
+        # update any newly installed libs to be relocatable
+        # this call is idempotent
+        ctx.local('virtualenv-2.7 --relocatable ../venv')
```

Comment:
I understand this call is supposed to be idempotent, and I *hope* this works out for us, but deploying virtualenvs this way still makes me nervous. In the meantime, hopefully if we run into any issues with this they'll show up in dev or stage first...